singleheader/CMakeLists.txt

```diff
@@ -113,8 +113,8 @@ add_test(amalgamate_demo amalgamate_demo ${EXAMPLE_JSON} ${EXAMPLE_NDJSON})
 
 install(FILES simdjson.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
-
-
+add_library(simdjson-singleheader STATIC "")
```

Comment:
This will build a "fake" library from the single-header files as a kind of test.

I guess we don't want to do 
```
 add_cpp_test(... COMPILE_ONLY)
```
?

Reply:
add_cpp_test is a good idea. amalgamate_demo should ideally be the same, and amalgamate directly from repo.

Reply:
Only real reason not to do it right now is, add_cpp_test isn't available in this directory. Not hard to make it so, though.